akka-stream/src/main/scala/akka/stream/impl/Throttle.scala

```diff
@@ -79,11 +113,8 @@ private[stream] class Throttle[T](cost: Int,
       override def onPull(): Unit = pull(in)
     })
 
-    override def preStart(): Unit = previousTime = now()
-
-    private def now(): Long = System.nanoTime()
+    override def preStart(): Unit = arrivalOfLastRecordedToken = System.nanoTime()
```

Comment:
It does slightly matter, since preStart is closer to actual startup. It is not that important though.